lib/orator/index.js

```diff
@@ -414,6 +420,14 @@ class Orator {
       `Hello! The prefix is \`${prefix}\`, try \`${prefix}help\``
     )
   }
+
+  async _missingWebhookPerms (response, bot, msg) {
+    const c = 'Sorry, this command requires the "Manage Webhooks" permission!'
+
+    await response.dm
```

Comment:
This doesn't not await the promise returned by the ternary. It awaits the property `response.dm`. You should probably use a regular if statement and return the promise instead of awaiting it

Reply:
above is correct, other example of this in the codebase wrapped the ternary in `()` also this logic is used twice so this could and should be made into a function